src/views/filters.rs

```diff
@@ -0,0 +1,16 @@
+use std::sync::OnceLock;
+
+use chrono::NaiveDateTime;
+
+static TZ: OnceLock<chrono_tz::Tz> = OnceLock::new();
```

Comment:
Unfortunately I don't think there's any better way of passing context to a filter function

Reply:
@swlody, could you please have a look if [runtime_values_in_custom_filters](https://github.com/askama-rs/askama/blob/6ff3b518f70c867aa1402930455b0a677c94bbd8/testing/tests/book_example_runtime_values_in_custom_filters.rs) is understandable? I think it might fit your usecase. The change is not merged, yet, though: https://github.com/askama-rs/askama/pull/397

Reply:
@Kijewski 

Sorry for not taking a look before you got the PR merged! In this very specific case I'm not sure how helpful this is, since we are only ever using a single timezone value (that of the application server) inside of the `format_datetime` filter.

"[runtime values](https://askama.readthedocs.io/en/stable/runtime.html) in custom filters" *would* very specifically help if we ever add the option to specify timezones for individual events, but our use case here is a website for a specific venue in a specific timezone, so I think the ergonomics of being able to render the template without calling `render_with_values()` outweighs the "code smell" of using a `OnceLock` to store the configured timezone.

@foltik for visibility

Reply:
@Kijewski appreciate you chiming in, loving Askama so far 😄 

We ended up just making the whole app config global: https://github.com/foltik/lsd/blob/main/src/utils/templates.rs#L6

Totally fine doing it this way, can't imagine any API changes in Askama that would make this easier. If anything we could have added a `config: &Config` field to each template and explicitly passed it to each filter that required accessing some field in the config, but chose this approach for less ceremony.